docker/Dockerfile

```
@@ -0,0 +1,57 @@
+
+# Protoc binary
+FROM node:lts-stretch as protoc-bin
+ARG PROTOC_VERSION
+RUN mkdir /installer
+RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip -O protoc.zip
+RUN mkdir -p /installer/protoc
+RUN unzip -o protoc.zip -d /installer/protoc/
+
+
+# Node
+FROM node:lts-stretch as node
```

```suggestion
FROM node:lts-alpine as node
```

29MB vs. 300MB

If this does not work try `node:lts-stretch-slim` (50MB)